File: 251229/출석 부르기 2/calling-attendance-2.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim();
const numberInput = input.split('\n').map(Number);
let i = 0;

while (true) {
    if (numberInput[i] === 1) {
        console.log('John');
    } else if (numberInput[i] === 2) {
        console.log('Tom');
    } else if (numberInput[i] === 3) {
        console.log('Paul');
    } else if (numberInput[i] === 4) {
        console.log('Sam');
    } else {
        console.log('Vacancy');
        break;
    }
    i += 1;
}

File: 260312/출석 부르기 2/calling-attendance-2.py
import sys

input = sys.stdin.readline

# 선언부
def student_list(n: int):
    if n == 1:
        return 'John'
    elif n == 2:
        return 'Tom'
    elif n == 3:
        return 'Paul'
    elif n == 4:
        return 'Sam'
    else:
        return 'Vacancy'

# 입력부
while True:
    i = int(input())
    name = student_list(i)
    print(name)
    if name == 'Vacancy':
        break
# 호출부
